### PR TITLE
Update osrd-website/content/about/use-case/strategic-timetable.xx.md

### PR DESCRIPTION
**Pull request by Christian-CROCHE**
All these explanation seem very France-oriented to me. Couldn’t these explanations be less chauvin?

### CODE REVIEW

**Review by axrolld — CHANGES_REQUESTED** [review]
One typo but the rest of the proposals seem good to me.
I don't know how the process works in other countries so i'm not very inspired on how to make this page "less French", but feel free to send other pull requests if you want to !

Thank you again for your contribution :)

**Comment by multun** [comment]
@Christian-CROCHE what do you think of https://github.com/DGEXSolutions/osrd-website/pull/80?

File: content/about/use-case/strategic-timetable.fr.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ weight: 10
...
+La première phase du processus de conception des plans de transport commence très en amont : 10&bnsp;ans (A-10) avant le jour de circulation. Elle a pour objectif **la proposition de plusieurs scenarios de circulation** sur des infrastructures ferroviaires existantes ou prospectives. Durant cette phase, SNCF Réseau (gestionnaire de l’infrastructure ferroviaire française et responsable de l’affectation et de la vente des sillons) collectionne les demandes des **AO** (aautorités organisatrices) et des **EF** (entreprises ferroviaires) en termes de mobilité. Les demandes sont transmises sous la forme de **schémas de dessertes** : représentation des trains demandés, leur itinéraire et les arrêts à desservir.
```
[the remark this answers is not in the document]
Comment (Christian-CROCHE) [reply content/about/use-case/strategic-timetable.fr.md:13]: Should I modify my proposal? I’m a rooky in GitHub and not used to its features…

**Comment by Christian-CROCHE** [comment]
> @Christian-CROCHE what do you think of #80? I didn't like these explanations either: they felt very dense and jargon-heavy, so I entirely rewrote the thing. Unfortunately, we somewhat forgot to integrate these changes. I would love to get your opinion on this take!

I’ll take some time to review this later. I’m currently following a training and thus, not really available.

- Should I review the two pages (English and French) like I did for the other pages?
- Are we supposed to write everything in English as we are mostly French, in order for all the other contributors to understand what we are saying?

**Comment by multun** [comment]
> I understand right now that the pull request#80 is already a modification proposal that you just « published », is that right?

That's it! All changes start as proposals, which can be reviewed, and integrated once all participants agree

> How should I propose you modifications on this?

There are a couple of ways we could do it:
 - We could also hop on a call and chat about it (You should have received a private message on your SNCF teams account)
 - Github has an integrated [review feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request). That's what we usually use, but it requires a little bit of getting used to.

> Are we supposed to write everything in English as we are mostly French, in order for all the other contributors to understand what we are saying?

That's the general idea, but not a hard rule either:
 - we try to communicate in english, as if a visible enough portion of the project were only available in french, it would make it unwelcoming to non-french contributors (which, believe it or not, are so far as interested as french contributors)
 - we don't want anyone to feel excluded either: if someone makes any kind of contribution in a language other than English, we'll handle it as usual

File: content/about/use-case/strategic-timetable.fr.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (axrolld) [reply content/about/use-case/strategic-timetable.fr.md:13]: Ideally yes, but we can also fix it. Are you using the Github interface to modify the files ?

Comment (Christian-CROCHE) [inline content/about/use-case/strategic-timetable.fr.md:13]:
```suggestion
La première phase du processus de conception des plans de transport commence très en amont : 10&bnsp;ans (A-10) avant le jour de circulation. Elle a pour objectif **la proposition de plusieurs scenarios de circulation** sur des infrastructures ferroviaires existantes ou prospectives. Durant cette phase, SNCF Réseau (gestionnaire de l’infrastructure ferroviaire française et responsable de l’affectation et de la vente des sillons) collectionne les demandes des **AO** (autorités organisatrices) et des **EF** (entreprises ferroviaires) en termes de mobilité. Les demandes sont transmises sous la forme de **schémas de dessertes** : représentation des trains demandés, leur itinéraire et les arrêts à desservir.
```

**Pull request closed without merging**